tools/api-markdown-documenter/src/api-item-transforms/default-implementations/TransformApiClass.ts

```diff
@@ -14,9 +14,9 @@ import {
 	type ApiProperty,
 } from "@microsoft/api-extractor-model";
 
-import { type SectionNode } from "../../documentation-domain/index.js";
-import { ApiModifier, filterByKind, isStatic } from "../../utilities/index.js";
-import { type ApiItemTransformationConfiguration } from "../configuration/index.js";
+import type { SectionNode } from "../../documentation-domain/index.js";
```

Comment:
Why move type outside the { } ?

Reply:
It actually has an impact on the runtime. Putting `type` on the outside of the braces results in the import statement being omitted from the generated JavaScript. If `type` is on the inside, then the import statement is persisted (and so the module is loaded at runtime), even if there are no runtime imports.

Reply:
I want to enable a linter rule for this case in our repo. Whenever possible, we should do full type-only import statements.